packages/contracts/mud.config.ts

```diff
@@ -72,7 +84,7 @@ export default mudConfig({
     {
       name: "UniqueEntityModule",
       root: true,
-      args: [],
+      args: [resolveTableId("RNGRequestOwner")],
```

Comment:
```suggestion
      args: [],
```
something went wrong